src/braket/emulation/emulator.py

```diff
@@ -79,11 +82,9 @@ def run(
             QuantumTask: The QuantumTask tracking task execution on this device emulator.
         """
 
-        task_specification = self.transform(task_specification, apply_noise_model=False)
-        # Don't apply noise model as the local simulator will automatically apply it.
-
+        task_specification = self.transform(task_specification, apply_noise_model=True)
         # Remove the verbatim box before submitting to the braket density matrix simulator
-        task_specification_v2 = self._remove_verbatim_box(task_specification)
+        task_specification_v2 = VerbatimTransformation().run(task_specification)
```

Comment:
Can "VerbatimTransformation()" be done as part of the "self.transform" above, meaning we append the "VerbatimTransformation()" into the modifier? See below.